### DISCUSSION TITLE
Wireframe: non-verbal communicator for means of inputting information

### DISCUSSION THREAD

**Discussion by GeorgiaHCA** [alan-turing-institute/AutSPACEs#368] [General]
### I would like to...

Have non-verbal means of inputting information to the platform 

### Because...

It enables non-verbal users to share their experiences

### Acceptance criteria

There are non-verbal means of inputting information to the platform
Which been tested with a range of users including non-verbal autistic users
Axure updated with the final UI design for the component
A picture/screenshot image of the final UI design of the component held within Axure
Justification of the component design available in Github repository as an md file - Fujitsu wireframes folder

### Other relevant documents:
Values
Ways of working
Data management
Focus group summaries

### Review owner:
Internal development team
Nonverbal members of the community
Alan Turing

---

## Notes

This user story was created by combining multiple insights from focus groups held in Autumn 2019. These focus groups were a mixture of autistic people, their families, and people from Autistica, the Turing, and Fujitsu. The data from the focus groups and more details about how it was collected and processed can be found here:

https://github.com/alan-turing-institute/AutisticaCitizenScience/tree/master/community-recommendations/focus-groups

Specifically, here: 

https://github.com/alan-turing-institute/AutisticaCitizenScience/blob/GeorgiaHCA-Summary1/community-recommendations/focus-group-summaries-2019.md#b-non-verbal-means-of-inputting-information

### Crisis logger

There's a nice example of a tool that allows multiple ways of inputting information at https://crisislogger.org

---

* Associated pull request: 
* Current status:

---

**Comment by ghost** [comment]
b) Non-verbal means of inputting information
(A d): “…having the option to…upload photos, or pictures, or voice recordings of people.”
(P g): “A lot of people do communicate quite visually, or understand things visually, so that would be useful, if people could have pictures”
(A p d): “communication with pictures, voice to text interface” to improve accessibility
c) Filters for different kinds of user
(A d): “that would be good…if you’re an autistic person who wants someone to relate to…you can always filter it through to see personal perspectives, whilst…parents might be looking for a different thing”
(R T d): “[maybe there is an] opportunity in this for autistic people and parents to empathise with each other through sharing experience…if you segregated the responses, it would prevent that from happening”
(P d): “you wouldn’t have to segregate them… you could just… have a drop box where people choose whether they’re parents, people with autism, teachers, employers…”
d) How should we ask the research question?
(A d): “…[make] sure the question’s not broad”
(A d): “…remind all people that this isn’t the experience of all of us, it’s just the experience of a good handful of us”
(R T d): “we’ll probably ask people to share a lot, and then it’s on the researcher, to…tease out the sensory processing aspect”
(R T i): “I think the only thing that I’m slightly worried about is if people feel that the experience that they’re sharing that is not related to the sensory processing doesn’t get the same level of attention, in the published research… [solution is]…setting expectations”
(A p d): “I think that when we collect several experiences for one particular environment, [researchers] can actually analyse that environment, without [participants] actually going into sensory analysis”

**Comment by ghost** [comment]
Crisis Logger

![crisis-logger](https://user-images.githubusercontent.com/41741006/85406270-75207680-b559-11ea-95b8-4a25dd39e9ad.PNG)

**Comment by GeorgiaHCA** [comment]
@Chris I like this a lot! And thanks for pulling out the evidence and discussion around the feature from the focus groups.

**Comment by ghost** [comment]
non-verbal communicator 


Wireframe Functionality 
https://c6q95y.axshare.com/#g=1&p=3-add-event-non-verbal-home-final


Dev Screens 
https://c6q95y.axshare.com/#g=1&p=10-non-verbal



Research 

https://c6q95y.axshare.com/#g=1&p=3-non-verbal-research
